src/Internal/Helpers.purs

```diff
@@ -282,3 +289,40 @@ concatPaths a b =
   right = fromMaybe b (stripPrefix (Pattern "/") b)
 
 infixr 5 concatPaths as <</>> -- </> is taken
+
+-- | Runs two `Aff` actions concurrently
+-- | Get resolved by the first which either resolves or throws
+-- | Expected properties:
+-- | race x y = race y x
+-- | race x (race y z) = race (race x y) z
+-- | race never x = x
+race :: forall (a :: Type). Aff a -> Aff a -> Aff a
```

Comment:
How does this differ from:

```purescript
race f g = sequential (parallel (try f) <|> parallel (try g)) >>= liftEither
```

Reply:
Indeed. This should be the same. Nice catch.